src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -111,6 +111,14 @@ protected override void OnPersist(ApplicationEngine engine)
             var pubkey = committee.OrderBy(p => p).ElementAt(index);
             var account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
             GAS.Mint(engine, account, gasPerBlock * CommitteeRewardRatio / 100);
+            if (engine.Snapshot.PersistingBlock.Index == 1)
+            {
+                // Genesis block reward
```

Comment:
Why we don't reward this in `OnInitialize` ? we only need to take care of not change the committee in the first block.

Reply:
Because you need to initialize GAS before NEO. But there is no way to control it in `OnInitialize()`.

Reply:
It's not enough with setting manually the order in this foreach?

https://github.com/neo-project/neo/blob/0f4244df193369e789bc60fa308a650b271673a6/src/neo/SmartContract/ApplicationEngine.Native.cs#L16-L24

Reply:
But moving it to `PostPersist()` is a easier way, isn't it?

Reply:
`PostPersist` can reduce the TPS because it will be executed in every block, but It's just a `if`

Reply:
I think that adding a function call to each block will have minimal impact on TPS.